squidb-tests/src/com/yahoo/squidb/test/TriggerTesterSpec.java

```diff
@@ -16,4 +16,8 @@
 
     @ColumnSpec(defaultValue = "0")
     int value2;
+
+    String str1;
+
+    String str2;
```

Comment:
Want to just make value1 and value2 text columns and use them for all the tests?